tests/test_mwan.py

```diff
@@ -304,7 +304,7 @@ def test_store_rule(e_uci, mocker):
     assert e_uci.get('mwan3', 'ns_rule_1', 'src_port') == '1:1024'
     assert e_uci.get('mwan3', 'ns_rule_1', 'dest_ip') == '10.0.0.2/12'
     assert e_uci.get('mwan3', 'ns_rule_1', 'dest_port') == '22,443'
-    assert e_uci.get('mwan3', 'ns_rule_1', 'sticky') == '0'
+    assert e_uci.get('mwan3', 'ns_rule_1', 'sticky') == '1'
```

Comment:
This is good, because the check is indeed on the configuration file rather than on the code